packages/api-docs-builder/utils/parseTest.ts

```diff
@@ -1,14 +1,18 @@
 import * as babel from '@babel/core';
-import { readFile, existsSync } from 'fs-extra';
+import { readFile } from 'fs-extra';
 import * as path from 'path';
+import glob from 'fast-glob';
 
 const workspaceRoot = path.join(__dirname, '../../../');
 const babelConfigPath = path.join(workspaceRoot, 'babel.config.js');
 
-function withExtension(filepath: string, extension: string) {
-  return path.join(
-    path.dirname(filepath),
-    path.basename(filepath, path.extname(filepath)) + extension,
+function getTestFilesNames(filepath: string) {
+  return glob.sync(
+    path.join(
+      path.dirname(filepath),
+      `/{tests/,}{*.,}${path.basename(filepath, path.extname(filepath))}.test.{js,ts,tsx}`,
+    ),
```

Comment:
Just the usual ;)

```suggestion
    path
      .join(
        path.dirname(filepath),
        `/{tests/,}{*.,}${path.basename(filepath, path.extname(filepath))}.test.{js,ts,tsx}`,
      )
      .replace(/\\/g, '/'),
```
(https://github.com/mrmlnc/fast-glob#how-to-write-patterns-on-windows)